ProcessLib/HeatTransportBHE/LocalAssemblers/HeatTransportBHELocalAssemblerSoil-impl.h

```diff
@@ -96,20 +106,36 @@ void HeatTransportBHELocalAssemblerSoil<ShapeFunction, IntegrationMethod>::
         auto const& M_ip = ip_data.NTN_product_times_w;
         auto const& K_ip = ip_data.dNdxTdNdx_product_times_w;
 
-        // auto const k_f = _process_data.thermal_conductivity_fluid(t, pos)[0];
-        // auto const k_g = _process_data.thermal_conductivity_gas(t, pos)[0];
-        auto const k_s = _process_data.thermal_conductivity_solid(t, pos)[0];
-
-        // auto const heat_capacity_f = _process_data.heat_capacity_fluid(t,
-        // pos)[0]; auto const heat_capacity_g =
-        // _process_data.heat_capacity_gas(t, pos)[0];
+        // auto const k_f =
+        // fluid_phase.property(MaterialPropertyLib::PropertyType::thermal_conductivity).template
+        // value<double>(vars, pos, t); auto const k_g =
+        // gas_phase.property(MaterialPropertyLib::PropertyType::thermal_conductivity).template
+        // value<double>(vars, pos, t);
+        auto const k_s =
+            solid_phase
+                .property(
+                    MaterialPropertyLib::PropertyType::thermal_conductivity)
+                .template value<double>(vars, pos, t);
+
+        // auto const heat_capacity_f =
+        // fluid_phase.property(MaterialPropertyLib::PropertyType::specific_heat_capacity).template
+        // value<double>(vars, pos, t); auto const heat_capacity_g =
+        // gas_phase.property(MaterialPropertyLib::PropertyType::specific_heat_capacity).template
+        // value<double>(vars, pos, t);
         auto const heat_capacity_s =
-            _process_data.heat_capacity_solid(t, pos)[0];
-
-        // auto const density_f = _process_data.density_fluid(t, pos)[0];
-        // auto const density_g = _process_data.density_gas(t, pos)[0];
-        auto const density_s = _process_data.density_solid(t, pos)[0];
-
+            solid_phase
+                .property(
+                    MaterialPropertyLib::PropertyType::specific_heat_capacity)
+                .template value<double>(vars, pos, t);
+
+        // auto const density_f =
+        // fluid_phase.property(MaterialPropertyLib::PropertyType::density).template
+        // value<double>(vars, pos, t); auto const density_g =
+        // gas_phase.property(MaterialPropertyLib::PropertyType::density).template
+        // value<double>(vars, pos, t);
```

Comment:
These "comments" were not touched for the last two years. I'd recommend to remove them.
If someday there is a need for those densities *etc.* it can be easily copied from the solid properties. No need to fix that dead "code"

Actually it was discussed in the original PR and nothing has changed: https://github.com/ufz/ogs/pull/2221#discussion_r231264509

Reply:
Although we will need them soon in order to include GW flow and partially saturated processes, they are not needed for now. Therefore removed.